### ISSUE TITLE
jspm update bug on dev dependencies

### ISSUE THREAD

**Issue by guybedford** [jspm/jspm-cli#766]
```javascript
jspm update babel

warn Did you mean jspm install babel@jspm:babel?

err  Update does not support explicit version ranges.
```

**Comment by guybedford** [comment]
Actually this isn't just for devDependencies.

**Comment by crisptrutski** [comment]
Fixed by 0c52dad6e85b9879ddba5d224047c52e3c1a6482

**Comment by guybedford** [comment]
Thanks!
